workflow.rst

```diff
@@ -378,11 +378,34 @@ order:
     * ``workflow.[workflow name].announce``
     * ``workflow.[workflow name].announce.[transition name]``
 
+    You can avoid triggering those events by using the context::
+
+        $workflow->apply($subject, $transitionName, [Workflow::DISABLE_ANNOUNCE_EVENT => true]);
+
+    .. versionadded:: 5.1
+
+        The ``Workflow::DISABLE_ANNOUNCE_EVENT`` constant was introduced in Symfony 5.1.
+
+    .. versionadded:: 5.2
+
+        The context will be accessible in all the events::
+
+            // $context must be an array
+            $context = ['context'];
+            $workflow->apply($subject, $transitionName, $context);
+
+            // in an event listener
+            $context = $event->getContext(); // returns ['context']
+
 .. note::
 
     The leaving and entering events are triggered even for transitions that stay
     in same place.
 
+.. note::
+
+    If you initialize the marking by calling ``$workflow->getMarking($object);``, then the ``workflow.[workflow name].entered.[initial place name]`` will be called with a default context ``Workflow::DEFAULT_INITIAL_CONTEXT``
```

Comment:
missing work "event" after the name of the event?

missing for at the end of the note.

Reply:
I'm sorry, I don't understand.

Where is the word "event" missing ?

And what's missing at the end of the note ?

Reply:
```rst
If you initialize the marking by calling ``$workflow->getMarking($object);``, then the ``workflow.[workflow name].entered.[initial place name]`` event will be called with a default context ``Workflow::DEFAULT_INITIAL_CONTEXT``.
```

Reply:
Indeed.

Thank you very much for spotting this :clap: